### PR TITLE
NAS-113470 / 22.02 / when saving a new license and reloading the UI does not reload

### PR DESCRIPTION
**Pull request by AlexKarpov98**

### CODE REVIEW

File: src/app/pages/system/general-settings/support/license/license.component.ts [hunk 1]
```diff
@@ -49,18 +49,20 @@ export class LicenseComponent {
...
-        hideCheckBox: true,
-        buttonMsg: helptext.update_license.reload_dialog_action,
-        hideCancel: true,
-        disableClose: true,
-      }).pipe(filter(Boolean), untilDestroyed(this)).subscribe(() => {
-        document.location.reload();
-      });
       this.slideInService.close();
       this.cdr.markForCheck();
+      setTimeout(() => {
```
Comment (denysbutenko) [inline src/app/pages/system/general-settings/support/license/license.component.ts:54]: Is there another possible approach to solve this without setTimeout?

Comment (denysbutenko) [reply src/app/pages/system/general-settings/support/license/license.component.ts:54]: For example, wrap `document.location.reload();` into `setTimeout()` or even add pipe `delay(200)`?

Comment (AlexKarpov98) [reply src/app/pages/system/general-settings/support/license/license.component.ts:54]: It does not work for me. While working the `slideInService` and `dialogService` together, the subscription was destroyed.
I think should run sequentially.
How do you think to close slide after confirmation?

**Review by denysbutenko — APPROVED** [review]
Works as described

**Comment by william-gr** [comment]
backport

**Pull request merged**